### PR TITLE
fix: correct error message when command list fails

### PR DESCRIPTION
**Pull request by maxrussell**
## Description

fix: correct error message when command list fails

Fix for [DET-7324]

## Test Plan

Running `det cmd list` now prints the following on error (and the output of `det tensorboard list` remains unchanged):

```
Master not found at localhost:8080. Hint: Remember to set the DET_MASTER environment variable to the correct Determined master IP or use the '-m' flag.
Failed to list commands: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105abbb20>: Failed to establish a new connection: [Errno 61] Connection refused'))
```

[DET-7324]: https://determinedai.atlassian.net/browse/DET-7324?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by maxrussell** [comment]
@cla-bot[bot] check

File: harness/determined/cli/notebook.py [hunk 1]
```diff
@@ -100,12 +101,12 @@ def open_notebook(args: Namespace) -> None:
             Arg("task_id", help="notebook ID", metavar="notebook_id"),
```
Comment (dzhu) [inline harness/determined/cli/notebook.py:101]: Looks like the `logs` command above this line should also get the same treatment.

File: harness/determined/cli/remote.py [hunk 2]
```diff
@@ -60,11 +61,11 @@ def run_command(args: Namespace) -> None:
...
-        Cmd("logs", lambda *args, **kwargs: task.logs(*args, **kwargs), "fetch command logs", [
+        Cmd("logs", partial(task.logs), "fetch command logs", [
             Arg("task_id", help="command ID", metavar="command_id"),
             *task.common_log_options,
         ]),
-        Cmd("kill", command.kill, "forcibly terminate a command", [
+        Cmd("kill", partial(command.kill), "forcibly terminate a command", [
             Arg("command_id", help="command ID", nargs=ONE_OR_MORE),
             Arg("-f", "--force", action="store_true", help="ignore errors"),
         ]),
```
Comment (dzhu) [inline harness/determined/cli/remote.py:71]: Also update `set_priority` below this.

**Review by dzhu — APPROVED** [review]
Hmm, seeing this is kind of making me think we could bear to move the wrapping into `declarative_argparse` itself, but I'm good with making this change for now. Thanks!

**Pull request merged**